execute/report/report.go

```diff
@@ -22,13 +21,11 @@ import (
 // The hasher and encoding codec are provided as arguments to allow for chain-specific formats to be used.
 //
 // The messages argument indicates which messages should be included in the report. If messages is empty
-// all messages will be included. This allows the caller to create smaller reports if needed. Executed messages
-// are skipped automatically.
+// all messages will be included. This allows the caller to create smaller reports if needed.
```

Comment:
Is it worth mentioning here that it is expected that messages have already been checked for execution validity? i.e this method is "blindly" proving and encoding a report.